build/params_mainnet.go

```diff
@@ -133,6 +133,7 @@ func init() {
 	if os.Getenv("LOTUS_DISABLE_DRAGON") == "1" {
 		UpgradeDragonHeight = math.MaxInt64 - 1
 		UpgradePhoenixHeight = math.MaxInt64
+		delete(DrandSchedule, UpgradePhoenixHeight)
```

Comment:
This _shouldn't_ be necessary. (it won't actually work given the order of operations?)

Reply:
actually, if I just move it up one line then we're all good and I can remove the drand.go hack, done that now